packages/client/components/TeamPrompt/Recurrence/RecurrenceSettings.tsx

```diff
@@ -251,7 +252,13 @@ export const RecurrenceSettings = (props: Props) => {
             freq: Frequency.WEEKLY,
             interval: recurrenceInterval,
             byweekday: recurrenceDays.map((day) => day.rruleVal),
-            dtstart: dayjs(recurrenceStartTime).utc().toDate(),
+            dtstart: datetime(
```

Comment:
+1 should we reuse method you created [here](https://github.com/ParabolInc/parabol/pull/8269/files#diff-021df5c0b52eff71f137510aa3223218ab5cf2d9ef15e1fb04e218210322965eR3)?

Reply:
Yep! Fixed